src/firefly/js/drawingLayers/ActiveTarget.js

```diff
@@ -27,11 +27,14 @@ var idCnt=0;
 
 
 function creator(initPayload, presetDefaults) {
-    var drawingDef= makeDrawingDef('magenta');
+    var drawingDef= makeDrawingDef('magenta',4 );
```

Comment:
the `4` is unnecessary. It takes an object, for example:
`var drawingDef= makeDrawingDef('magenta', 
{lineWidth:4, size:6, 
fontWeight:'bolder', 
   symbol: DrawSymbol.CIRCLE } );`